test/data/test-ini

```diff
@@ -21,7 +21,9 @@
 [string]
 	simple = A simple string
 	quoted = "A quoted string"
-	quoted_with_quotes = "A string "with quotes""
+	quoted_with_quotes = "A string \"with quotes\""
+	quoted_with_comment = "A quoted string" # a comment
+	escaped_quotes = String \"with quotes\"
```

Comment:
> * Unknown escape sequences

That would cement the lax behavior.  So I guess we should discuss it: Do you like the lax behavior of treating an unrecognized escape sequence as verbatim?  There's something to be said for strictness, and having that be an error.